lld/wasm/Driver.cpp

```diff
@@ -317,10 +317,6 @@ void LinkerDriver::addFile(StringRef path) {
     if (inWholeArchive) {
       for (const auto &[m, offset] : members) {
         auto *object = createObjectFile(m, path, offset);
-        // Mark object as live; object members are normally not
-        // live by default but -whole-archive is designed to treat
-        // them as such.
-        object->markLive();
```

Comment:
This was unnecessary because the forth argument to `createObjectFile` (`lazy`) defaults to `false` which then calls `markLive`.